core/tests/puppeteer-acceptance-tests/user-utilities/voiceover-admin-utils.ts

```diff
@@ -55,10 +57,19 @@ export class VoiceoverAdmin extends BaseUser {
     } else {
       await this.clickOn(explorationSettingsTab);
     }
+
+    showMessage('Navigation to settings tab is successfull.');
```

Comment:
"successful" is misspelled, please fix here and elsewhere

Reply:
done